packages/fxa-settings/src/lib/oauth/oauth-errors.ts

```diff
@@ -16,93 +17,93 @@ export const UNEXPECTED_ERROR = 'Unexpected error';
 
 export const OAUTH_ERRORS: Record<string, AuthError> = {
   UNKNOWN_CLIENT: {
-    errno: 101,
+    errno: OAUTH_ERRNO.UNKNOWN_CLIENT,
     message: 'Unknown client',
```

Comment:
This string is repeated in `libs/accoutns.errs/src/oauth-error.ts`. Can clean this up here, either by referencing the error directly, or backfilling this object with properites from the source error type. The latter is probably safer and more backwards compatible, eg:

```
UNKNOWN_CLIENT: (() => { 
    const err = OauthError.unknownClient(''); 
    return {
        errno: err.errno,
        message: err.message
    },
})(),
```


On a side note, I noticed the errno values in `libs/accounts/src/oauth-error.ts` are hard coded. They should reference the `OAUTH_ERRNO` definitions in `constants.ts`.